src/components/pacbio/PacbioPoolTubeItem.vue

```diff
@@ -8,35 +8,51 @@
   >
     <b-row>
       <b-col cols="3">
-        <b-img src="/tube.png" :draggable="!!valid" />
+        <b-img src="/tube.png" />
+        <b-button
+          :id="`editPool-${id}`"
+          size="sm"
+          variant="outline-primary"
+          class="btn-block"
+          :to="{ name: 'PacbioPoolCreate', params: { id: id } }"
+          >Edit</b-button
+        >
```

Comment:
If a pool is not suitable for run creation, the user may wish to quickly fix it. This prevent them from needing to pop back to the pool page to find it, and can instead take them directly to the edit page.